src/java/org/apache/cassandra/config/Config.java

```diff
@@ -264,6 +264,8 @@
 
     public boolean inter_dc_tcp_nodelay = true;
 
+    public boolean enable_ssl_nte = true;
```

Comment:
prefer making changes in cassandra_env.sh rather than Config.java.

Reason being that Config.java changes are breaking changes that make rolling versions harder, whereas cassandra_env.sh will just no-op if the variable doesn't exist.